src/components/calcite-fab/calcite-fab.stories.ts

```diff
@@ -0,0 +1,56 @@
+import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
+import {
+  Attributes,
+  createComponentHTML as create,
+  darkBackground,
+  parseReadme,
+  titlelessDocsPage
+} from "../../../.storybook/utils";
+import readme from "./readme.md";
+import { ATTRIBUTES } from "../../../.storybook/resources";
+const { dir, theme } = ATTRIBUTES;
+
+export default {
+  title: "components|calcite-fab",
+  decorators: [withKnobs],
+  parameters: {
+    backgrounds: darkBackground,
+    docs: {
+      page: titlelessDocsPage
+    },
+    notes: parseReadme(readme)
+  }
+};
+
+const createAttributes: () => Attributes = () => [
+  {
+    name: "dir",
+    value: select("dir", dir.values, dir.defaultValue)
+  },
+  {
+    name: "disabled",
```

Comment:
@asangma can we add `appearance`, `scale` and `icon` here to the story now that its added to the component?

Reply:
Oh yeah.

Reply:
Added. 👍